common-design/templates/cd-header.html

```diff
@@ -247,5 +245,15 @@
     e.stopPropagation();
     e.preventDefault();
   });
+
+  $('.cd-search__input').on('focus', function(e){
+    $(this).parent().find('.cd-search__submit').toggleClass('js-has-focus');
+  });
+
+  $('.cd-search__input').on('blur', function(e){
+    $(this).parent().find('.cd-search__submit').toggleClass('js-has-focus');
+  });
+
+
```

Comment:
I would prefer to store our JS in separate JS files and avoid embedding scripts within HTML templates. The `<script>` tag should appear just before the `</body>` closing tag. Additionally, we will avoid common jQuery problems by wrapping any JS we write in an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) like so:

```js
(function($) {
  // Inside this function block, the $ represents jQuery even when
  // it was initialized in noConflict mode, the default setting in D7
})(jQuery);
```

Plus one additional UX consideration.. should the box gain focus automatically when toggling visibility by clicking on the nav button? I do notice clicking then hitting <kbd>Tab</kbd> brings focus to search input.